src/goto-instrument/contracts/dynamic-frames/dfcc_contract_mode.h

```diff
@@ -13,11 +13,15 @@ Date: August 2022
 #ifndef CPROVER_GOTO_INSTRUMENT_CONTRACTS_DYNAMIC_FRAMES_DFCC_CONTRACT_MODE_H
 #define CPROVER_GOTO_INSTRUMENT_CONTRACTS_DYNAMIC_FRAMES_DFCC_CONTRACT_MODE_H
 
+#include <string>
+
 /// Enum type representing the contract checking and replacement modes.
 enum class dfcc_contract_modet
 {
   CHECK,
   REPLACE
 };
 
+std::string dfcc_contract_mode_to_string(const dfcc_contract_modet mode);
```

Comment:
How about defining `operator<<` instead? I don't really know what the use cases look like.

Reply:
sometimes we need to produce an actual string to construct an exception.

Reply:
added `operator<<`